gqcp/include/Molecule/Molecule.hpp

```diff
@@ -132,6 +134,19 @@ class Molecule {
      */
     int charge() const;
 
+    /**
+     *  @return a textual description of this molecule.
+     */
+    std::string description() const {
+
+        std::string molecule_string = (boost::format("Number of electrons: %s \n") % this->numberOfElectrons()).str(); 
+        std::ostringstream framework;
+        framework << this->nuclearFramework();
```

Comment:
Could you also refactor this into `NuclearFramework::description()`?

Reply:
Allright, will do!